README.md

```diff
@@ -1 +1,35 @@
-# OOP_school_library
\ No newline at end of file
+# oop_library_school
+
+is a ruby program that allows the user to to record what books are in the library and who borrows them. The app that you will create will allow you to:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when
+
+## Getting Started
+To setup the project use :
+git clone https://github.com/nedjwab/OOP_school_library.git
+
```

Comment:
- [OPTIONAL] Please it will be nice to add a `setup section` with `steps` to clone and run your app locally, as this will be of help to none technical users who would like to know how your app works.